usr/share/rear/build/default/980_verify_rootfs.sh

```diff
@@ -1,18 +1,50 @@
-# make sure that the rootfs contains a usable system
+# Test that the ROOTFS_DIR contains a usable system
+# i.e. test that the ReaR recovery system will be usable
+# to avoid issues like https://github.com/rear/rear/issues/1494
 
-# In case the filesystem is mounted 'noexec' we skip a chroot bash test
-tmp_filesystem=$(filesystem_name $ROOTFS_DIR)
+# In case the filesystem that contains the ROOTFS_DIR is mounted 'noexec' we must skip the 'chroot' tests.
+# The filesystem_name function in linux-functions.sh returns the mountpoint (not a filesystem name like 'ext4'):
+rootfs_dir_fs_mountpoint=$( filesystem_name $ROOTFS_DIR )
 
-if grep -qE '^\S+ '$tmp_filesystem' \S+ \S*\bnoexec\b\S* ' /proc/mounts; then
-	LogPrint "WARNING: Filesystem $tmp_filesystem is mounted 'noexec', aborting chroot bash test"
-	Log "The above error means that we cannot test our chrooted environment because
+if grep -qE '^\S+ '$rootfs_dir_fs_mountpoint' \S+ \S*\bnoexec\b\S* ' /proc/mounts ; then
+    LogPrint "Cannot test if the ReaR recovery system is usable because $rootfs_dir_fs_mountpoint is mounted 'noexec'"
+    Log "We cannot run tests inside the ReaR recovery system by using 'chroot' because
```

Comment:
Maybe one `LogPrint` line with the full explanation is good enough. After all, this is a serious issue. I would even consider failing here with `Error` and to make it mandatory for ReaR to really check the rescue image. IMHO it is much more important to guarantee a runnable rescue image than to support `noexec` environments. For those we could also add a new feature (not in this PR) for ReaR to create a `tmpfs` to work in. I guess nowadays most servers have enough RAM for that.

Reply:
I was also wondering about that explanation via 'Log'.
I removed that and exit with an 'Error' now.